beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,486 @@ title: Quick Start
 
-<LearnMore path="/learn/describing-the-ui">
+```js
+<div>
+  {isLoggedIn && <AdminPanel />}
+</div>
+```
 
-Read **[Describing the UI](/learn/describing-the-ui)** to learn how to make things appear on the screen, including declaring components, importing them, writing JSX with the curly braces, and writing conditions and lists.
+If you're unfamiliar with this JavaScript syntax, you can start by always using `if...else`.
 
-</LearnMore>
+## Rendering lists {/*rendering-lists*/}
 
-### Chapter 2 overview: Adding interactivity {/*chapter-2-overview-adding-interactivity*/}
+You will rely on JavaScript features like [`for` loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for) and the [array `map()` function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to render lists of components.
 
-Components often need to change what’s on the screen as a result of an interaction. Typing into the form should update the input field, clicking “next” on an image carousel should change which image is displayed, clicking “buy” puts a product in the shopping cart. Components need to “remember” things: the current input value, the current image, the shopping cart. In React, this kind of component-specific memory is called [*state*](/learn/state-a-components-memory).
+For example, let's say you have an array of products:
 
-You can add state to a component with a [`useState`](/reference/usestate) Hook. Hooks are special functions that let your components use React features (state is one of those features). The `useState` Hook lets you declare a state variable. It takes the initial state and returns a pair of values: the current state, and a state setter function that lets you update it.
+```js
+const products = [
+  { title: 'Cabbage', id: 1 },
+  { title: 'Garlic', id: 2 },
+  { title: 'Apple', id: 3 },
+];
+```
 
-This `Gallery` component needs to remember two things: the current image index (initially, `0`), and whether the user has toggled "Show details" (initially, `false`):
+Inside your component, use the `map()` function to transform an array of products into an array of `<li>` items:
 
 ```js
-const [index, setIndex] = useState(0);
-const [showMore, setShowMore] = useState(false);
+const listItems = products.map(product =>
+  <li key={product.id}>
+    {product.title}
+  </li>
+);
+
+return (
+  <ul>{listItems}</ul>
+);
 ```
 
-Notice how clicking the buttons updates the screen:
+Notice how `<li>` has a `key` attribute. A key helps React figure out what happened when list items are inserted, removed, or reordered. Usually you should use a key from your data, such as a database ID. Keys should be unique among items in the same parent tag. Although you can technically specify index as a key, your list won't work correctly when items are inserted, removed, or reordered. Always try to use ID as a key if you can.
```

Comment:
> Although you can technically specify index as a key, your list won't work correctly when items are inserted, removed, or reordered. Always try to use ID as a key if you can.

Would probably rewrite this as "Do not use an index as a key, because..." since I don't think there's ever a good reason to do so, even if they technically can.

Reply:
I don't think it's that simple in practice. I do regularly bump into cases where index is the right key. If we push people towards "index always bad" this can be na issue, e.g. they can use random keys which are even worse. I'll try to rephrase this to be more concise though.